DEVELOPMENT.md

```diff
@@ -0,0 +1,60 @@
+# Development
+
+Thank you for your interest in developing on TypeScript ESLint! ❤️‍🔥
+
+> See [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our general contribution flows.
+
+## Setup
+
+After [forking the repo from GitHub](https://help.github.com/articles/fork-a-repo):
+
+```shell
+git clone https://github.com/<your-name-here>/typescript-eslint
+cd typescript-eslint
+yarn
+yarn build
+```
+
+## Validating Changes
```

Comment:
Do we want to mention the integration tests? It should be pretty rare that someone needs to run them manually - so probably not?

Reply:
I don't think I can remember ever running them... let's say not?